docs/tables/aws_lambda_function_event_source_mapping.md

```diff
@@ -0,0 +1,61 @@
+# Table: aws_lambda_function_event_source_mapping
+
+AWS Lambda is a compute service that lets you run code without provisioning or managing servers.
+Event Source Mappings are triggers that route events from services like Kafka, Kinesis, SQS or SNS to the configured AWS Lambda.
+
+## Examples
```

Comment:
Are there any other examples we can add that highlight which fields are relevant to specific resource types? For instance, looking at https://docs.aws.amazon.com/lambda/latest/dg/API_GetEventSourceMapping.html#API_GetEventSourceMapping_ResponseElements, a lot are only relevant to DynamoDB streams, so if there was an example only listing event source mappings using DDB streams and had DDB relevant columns in its `select` statement, that would be easy for users to run and know which columns are relevant.